.github/workflows/sync-upstream-push.yml

```diff
@@ -0,0 +1,33 @@
+name: Sync with Upstream (push)
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    if: github.event.review.state == 'approved' && startsWith(github.event.pull_request.head.ref, 'sync-upstream-')
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Configure Git
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+
+      # TODO: Change target branch to main once the workflow is ready
+      - name: Push changes to main and delete branch
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
is this needed when you're only running `git` directly?

Reply:
Oh, this is left over from when this used the CLI, you're right! Will remove!

Reply:
Done!